go_lib/dependency/k8s/drain/helper.go

```diff
@@ -36,5 +47,27 @@ func NewDrainer(kubeClient kubernetes.Interface) *Helper {
 		ErrOut:  io.Discard,
 	}
 
+	if config.Force != nil {
+		drainer.Force = *config.Force
+	}
+	if config.IgnoreAllDaemonSets != nil {
+		drainer.IgnoreAllDaemonSets = *config.IgnoreAllDaemonSets
+	}
+	if config.DeleteEmptyDirData != nil {
+		drainer.DeleteEmptyDirData = *config.DeleteEmptyDirData
+	}
+	if config.GracePeriodSeconds != nil {
+		drainer.GracePeriodSeconds = *config.GracePeriodSeconds
+	}
+	if config.Timeout != nil {
+		drainer.Timeout = *config.Timeout
+	}
+	if config.Out != nil {
```

Comment:
config.Out is an Interface and comparing with nil may have side effect, that variable is not nil but interface data is nil

Reply:
Actually these parameters are not necessary here, I just remove them